partcad/src/partcad/runtime_python_conda.py

```
@@ -80,19 +82,24 @@
+                        ]
+                        span.set_attribute("cmd", " ".join(args))
+                        # Install new conda environment with the preferred Python version
+                        p = subprocess.Popen(
+                            args,
+                            stdout=subprocess.PIPE,
+                            stderr=subprocess.PIPE,
+                            shell=False,
+                            encoding="utf-8",
+                        )
+                        _, stderr = p.communicate()
                     if not stderr is None and stderr.strip() != "":
```

Ruff (0.8.2)

Test for object identity should be `is not`

Convert to `is not`

(E714)